### PR TITLE
docs(): add security policy for kubeslice-cli

### PR DESCRIPTION
**Pull request by Himan-Miku**
# Description
Added SECURITY.md file to the repository to provide guidelines and procedures for reporting security vulnerabilities. This file outlines how security issues should be reported and handled within the project.

## How Has This Been Tested?
No explicit tests are needed for this change. The file addition is tested implicitly by ensuring it appears in the repository structure.

## Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR requires documentation updates? (Not applicable for this change)
* [ ] I've updated documentation as required by this PR. (Not applicable for this change)
* [x] I have performed a self-review of my own code.
* [ ] I have commented my code, particularly in hard-to-understand areas. (Not applicable for this change)
* [ ] I have tested it for all user roles. (Not applicable for this change)
* [ ] I have added all the required unit test cases. (Not applicable for this change)

## Does this PR introduce a breaking change?

```release-note

```

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Thank you for your interest in the security of the kubeslice project. We take security seriously, and we appreciate your assistance in identifying and disclosing security vulnerabilities responsibly.
+
+## Reporting a Security Vulnerability
+
+If you discover a security vulnerability in the kubeslice project, please follow these steps to report it:
+
+1. **Privately** disclose the vulnerability to the project maintainers. You have two options:
+
+   - Email us at [security@kubeslice.io](mailto:security@kubeslice.io).
+   - Create a new security advisory on the [Security Advisories](https://github.com/kubeslice/kubeslice/security/advisories) page.
```
Comment (narmidm) [inline SECURITY.md:12]: change url from https://github.com/kubeslice/kubeslice/security/advisories this to 
https://github.com/kubeslice/kubeslice-cli/security/advisories

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+2. Provide a detailed description of the vulnerability, including information about the affected versions of the project.
+
+3. We will acknowledge your report within 48 hours, and we will work with you to verify and address the issue promptly.
+
+4. Once the vulnerability is confirmed and mitigated, we will work on releasing a fix. We will credit you for your responsible disclosure if you desire.
+
+## Security Advisories
+
+To view the latest security advisories for the kubeslice Charts project, please visit the [Security Advisories](https://github.com/kubeslice/kubeslice/security/advisories) page.
```
Comment (narmidm) [inline SECURITY.md:22]: change for the kubeslice Charts project -> for the kubeslice cli project

change url from https://github.com/kubeslice/kubeslice/security/advisories this to 
https://github.com/kubeslice/kubeslice-cli/security/advisories

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## Supported Versions
+
+The kubeslice Charts project is actively maintained, and we typically provide security updates for the latest release and the previous release. Users are encouraged to keep their installations up to date with the latest releases to ensure they are protected against known vulnerabilities.
```
Comment (narmidm) [inline SECURITY.md:26]: change for the kubeslice Charts project -> for the kubeslice cli project

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+## Security Best Practices
+
+While we strive to maintain a secure codebase, users and contributors can help by following these security best practices:
+
+- **Regularly Update**: Keep your kubeslice Charts installation and its dependencies up to date to benefit from the latest security fixes.
```
Comment (narmidm) [inline SECURITY.md:32]: change for the kubeslice Charts project -> for the kubeslice cli project

File: SECURITY.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+- **Implement Access Controls**: Limit access to the kubeslice and its resources only to authorized personnel and entities. Employ Kubernetes RBAC (Role-Based Access Control) and other access control mechanisms.
+
+- **Scan for Vulnerabilities**: Use security scanning tools to regularly scan your Kubernetes clusters and the kubeslice for potential vulnerabilities.
+
+- **Audit Logs**: Enable and regularly review logs for the kubeslice and your Kubernetes environment to identify any suspicious activities.
+
+- **Third-party Dependencies**: Be cautious when using third-party libraries or tools in your deployment and keep them updated as well.
+
+- **Educate Team Members**: Ensure that your team is educated about security best practices and follows them when working with kubeslice Charts.
```
Comment (narmidm) [inline SECURITY.md:42]: change for the kubeslice Charts project -> for the kubeslice cli project

File: SECURITY.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+## Attribution
+
+We would like to thank the security researchers and contributors who have helped improve the security of the kubeslice Charts project by responsibly disclosing security vulnerabilities.
```
Comment (narmidm) [inline SECURITY.md:46]: change for the kubeslice Charts project -> for the kubeslice cli project

File: SECURITY.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+## Contact Us
+
+If you have any questions or concerns regarding this security policy or the security of the kubeslice Charts project, please contact us at [security@kubeslice.io](mailto:security@kubeslice.io).
```
Comment (narmidm) [inline SECURITY.md:50]: change for the kubeslice Charts project -> for the kubeslice cli project

**Comment by Himan-Miku** [comment]
ok

**Comment by Himan-Miku** [comment]
Hey @narmidm can you please review the changes? and If I've done anything wrong can you please guide me?

**Comment by narmidm** [comment]
> Hey @narmidm can you please review the changes? and If I've done anything wrong can you please guide me?

Looking good to me. I am approving & merging it out. 

and @Himan-Miku,
👍 Fantastic work on your first PR! 🚀 Keep up the good work and continue raising PRs.

**Review by narmidm — APPROVED** [review]

**Comment by Himan-Miku** [comment]
Yeah sure 👍

**Pull request merged**